src/System.Diagnostics.Process/ref/4.0/System.Diagnostics.Process.Manual.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Adding @ericstj . @stephentoub files in src\System.Diagnostics.Process/ref refers to the latest contract under development. For most purposes that will be the only reference that will be in our tree. However in some cases we might have 2 different version of the unstable contracts under development, For example: Process and NetworkInformation in which case the other unstable contracts are under there own specific versions hence ref/4.0